apps/core/src/pages/index.tsx

```diff
@@ -61,15 +62,29 @@ export const Component = () => {
     <>
       <div
         style={{
-          width: 300,
-          margin: '80px auto',
-          borderRadius: '8px',
-          boxShadow: 'var(--affine-shadow-2)',
-          backgroundColor: 'var(--affine-background-overlay-panel-color)',
-          padding: '16px 12px',
+          position: 'fixed',
+          left: '50%',
+          top: '50%',
         }}
       >
-        <UserWithWorkspaceList />
+        <Menu
```

Comment:
why we need use menu to render a page

Reply:
I think this is not @joooye34 was asking. It looks like we need to design a page for no workspace - or perhaps this should not happen because we will create one when there isn't any for first visit?

Reply:
This situation arises when you’ve removed all workspaces. It used to be that you couldn’t delete the last workspace, but it seems that due to some product design changes, it’s allowed now, isn’t it? @houjoe0829

Reply:
https://github.com/toeverything/AFFiNE/issues/4455
I’ve transitioned this issue over to GitHub, let’s no longer continue the discussion on this functionality here.